### ISSUE TITLE
CartPriceRule for free shipping will be just applied on second time cart update

### ISSUE THREAD

**Issue by albertmueller** [coreshop/CoreShop#698]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no

Can be simulate by following steps:
1. Add a cart price rule -> price from EUR 99 to 99999 for all carriers and use as action free shipping.
2. Add products in cart below EUR 99 -> shipping costs will be applied
3. Change the qty that you have more as EUR 99 as summary -> shipping costs stay alive
4. Click on cart refresh -> now it's free shipping
5. Change the qty back, below EUR 99 -> free shipping stay alive
6. Click on cart refresh -> shipping costs will be applied
So in summary, just on second time i got the right values.

**Comment by dpfaffenbauer** [comment]
fixed and merged
